software.rst

```diff
@@ -31,3 +31,11 @@ Data warehouse for high-powered scenario analysis in the domain of
 integrated assessment of climate change and energy systems modeling
 
 https://docs.ece.iiasa.ac.at/projects/ixmp4
+
+NUTS utility: **pysquirrel**
+-------------------------
+
+Python package designed to work with NUTS administrative divisions, searching
```

Comment:
You could make it a bit more of a complete sentence.
```suggestion
Pysquirrel is a python package designed to work with NUTS administrative divisions, searching
```

and then below:
"For more information on how to install and use the package please refer to the README on GitHub".